lib/controllers/prepare-controller.ts

```diff
@@ -114,12 +115,16 @@ export class PrepareController extends EventEmitter {
 			return false;
 		}
 
+		const pluginsNativeDirectories = this.$nodeModulesDependenciesBuilder.getProductionDependencies(projectData.projectDir)
+			.filter(dep => dep.nativescript)
+			.map(dep => path.join(dep.directory, "platforms", "ios"));
```

Comment:
platforms and ios are available in constants (I think), so you can use them from there. Ensure you've used `toLowerCase()` of the iOS constant.